src/main/java/juja/sqlcmd/Application.java

```diff
@@ -1,6 +1,153 @@
 package juja.sqlcmd;
 
+import java.sql.*;
+
 public class Application {
-    public static void main(String[] args) {
+
+    private static final String POSTGRESQL_DRIVER = "org.postgresql.Driver";
+    private static final String DB_CONNECTION_PATH = "jdbc:postgresql://127.0.0.1:5432/sqlcmd";
+    private static final String DB_USER = "sqlcmd";
+    private static final String DB_PASSWORD = "sqlcmd";
+    private static final String TABLE_NAME = "user";
+    private static final String LINE_SEPARATOR = System.lineSeparator();
+    private Connection connection;
+
+
+    public static void main(String[] arg) throws SQLException {
+        new Application().simpleSQL();
+    }
+
+    public void simpleSQL() throws SQLException {
+        checkJDBCDriver();
+        createDbConnection();
+        dropTableWithName(TABLE_NAME);
+        showExistingTablesName();
+        createTable(TABLE_NAME);
+        for (int i = 1; i < 4; i++) {
+            addUser("user" + i, "password" + i);
+        }
+        showExistingTablesName();
+        showExistingRecordsOfTable(TABLE_NAME);
+        updateDataByTitle("user2", "userchange1", TABLE_NAME, "name");
```

Comment:
Why do you pass in as a parameter tableName to update method but don't do it with addUser? I think 4 parameters for method is pretty much. https://goo.gl/7cVP2j